nxdrive/commandline.py

```diff
@@ -577,12 +579,13 @@ def launch(self, options: Optional[Namespace], /, *, console: bool = False) -> i
                 log.warning(f"{APP_NAME} is already running: exiting.")
             return 0
 
+        exit_code: int = 1
```

Comment:
Does `mypy` throw a warning/error and you had to fix it like that?

Reply:
Not really, my handling of the exception in the `HealthCheck.__exit__()` made it so that the code after `exit_code = app.exec_()` is still called in case of a crash. I had an error since in case of a crash, `exit_code` is undefined.

Reply:
And the `: int` was already there before